eth-rpc-adapter/src/transports/http.ts

```diff
@@ -53,7 +56,19 @@ export default class HTTPServerTransport extends ServerTransport {
     logger.debug(req.body, 'incoming request');
     let result = null;
     if (req.body instanceof Array) {
-      result = await Promise.all(req.body.map((r: JSONRPCRequest) => super.routerHandler(r)));
+      if (req.body.length > this.options.batch_size) {
+        const error = new InvalidRequest();
+        result = {
+          jsonrpc: '2.0',
+          error: {
+            code: error.code,
+            data: error.data,
+            message: 'Exceeded maximum batch size'
```

Comment:
```suggestion
            message: `Exceeded maximum batch size ${this.options.batch_size}`
```

Reply:
I think we can give user more detail so they won't be confused about "what is the maximum size?"